style.css

```diff
@@ -1,3 +1,230 @@
+	padding: 20px;	
+}
+
+nav {
+	display: flex;
+	margin: 20px auto;
+	gap: 20px;
+	justify-content: space-evenly;
+}
+
+nav a {
+	font-size: 2.5rem;
+}
+
+
+/* INTRODUCTION/WELCOME SECTION */
+
+.introduction_section {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-end;
+	justify-content: center;
+	column-gap: 20px;
+	width: 100%;
+	height: 900px;
+	background-image: url(images/horse_head_nebula.jpg); /* a place-holder image */ 
+	background-size:cover; /* allows to get the background in the whole section*/
+}
+
+.introduction_section h2 {
+	color: white;
+	font-size: 5rem;
+	font-variant: small-caps;
+	margin-left: 30%;
+	margin-right: 15%;
+}
+
+.introduction_section p {
+	background-color: rgba(0,0,0,0.5);
+	color: white;
+	margin-left: 30%;
+	margin-right: 15%;	
+	font-size: 3.5rem;
+	font-variant: small-caps;	
+}
+
+
+/* ABOUT ME SECTION */
+
+.aboutMe_section {
+	display: flex;
+  flex-direction: column;
+	align-items: flex-start;
+	justify-content: center;
+	background-color: #262323;	
+}
+
+.aboutMe_card {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	align-items: center;
+	margin-left: 50px;
+	gap: 75px;
+}
+
+.aboutMe_section img {
+	width: 200px;
+	height: 250px;
+	border-radius: 10px;
+}
+
+.aboutMe_text {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start ;		
+	max-width: 1000px;
+}
+
+.aboutMe_text p, .aboutMe_text a {
+	font-size: 2.4rem;
+}
+
+.aboutMe_text a {
+	color: white;
+	background-color: #d65a64;
+	padding: 5px;
+	border-radius: 10px;
+}
+
+/* PORTFOLIO SECTION */ 
+
+.portfolio_section {
+	background-color: #171616;
+}
+
+.portfolio_section h2 {
+	margin: 0 50px;
+}
+
+.project_cards_container {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	margin: 20px 50px;
+	gap: 50px 200px;
+}
+
+.project_card {
+	border: 2px solid #262323;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-starts;
+	max-width: 400px;
+	padding: 20px;
+	border-radius: 10px;
+	column-gap: 50px;
+
+}
+
+.project_card img, .project_card p {
+	height: 200px;
+}
+
+.project_links_bar {
+	display: flex;
+	justify-content: space-between;
+}
+
+
+/* CONTACT AMD FOOTER SECTIONS */
+					
+.contact_section{
+	margin: 0;
+	background-color: #262323;	
+}		
+
+.contact_section p{
+	margin: 50px 50px;
+	font-size: 2.4rem;
+}			
+					
+.contact_logos_bar {
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	flex-wrap: wrap;
+	gap: 50px;
+	padding-left: 50px;
+}								
+
+.contact_logos_bar img { 
+	height: 100px;
+}
+
+footer {
+	margin: 0;
+	padding: 20px;
+	background-color: #171616;	
+	
+}
+
+/* A FEW ADAPTATIONS FOR MOBILE DISPLAY */
+
+@media screen and (max-width:800px) {  
+/*On the mobile version, there will be no navbar*/
+	nav{
+		display: none;	
+	}
+		
+	.introduction_section h2, .introduction_section p, .aboutMe_card, .aboutMe_section h2, .portfolio_section h2{
+	 	justify-content: center;
```

Comment:
out of all the above elements, `justify-content` is only a valid property for `.aboutMe_card`

Reply:
I removed it